cmd/fyne_demo/main.go

```diff
@@ -22,8 +22,7 @@ func formApp(app fyne.App) {
 	email := widget.NewEntry()
 	email.SetPlaceHolder("test@example.com")
 	password := widget.NewPasswordEntry()
-	largeText := widget.NewEntry()
-	//	largeText.Text = "\n\n\n"
```

Comment:
I left this commented out as it currently breaks the form layout - unless you have another solution for that locally? :)

Reply:
Apologies, that is what I was referring to

Reply:
Added a FormLayout in 285531e883586687ad283c720140ec465c8916ce

Reply:
FormLayout feature has been reverted and moved to https://github.com/fyne-io/fyne/pull/72